ocaml/parsing/pprintast.ml

```diff
@@ -602,14 +603,15 @@ and label_exp ctxt f (l,opt,p) =
                  (pattern1 ctxt) p (expression ctxt) o
            | None -> pp f "?%s:%a" rest (maybe_local_pat ctxt is_local) p)
       end
-  | Labelled l -> match p with
+  | Labelled l -> (match p with
     | {ppat_desc  = Ppat_var {txt;_}; ppat_attributes = []}
       when txt = l ->
         if is_local then
           pp f "~(local_ %s)" l
         else
           pp f "~%s" l
-    | _ ->  pp f "~%s:%a" l (maybe_local_pat ctxt is_local) p
+    | _ ->  pp f "~%s:%a" l (maybe_local_pat ctxt is_local) p)
+  | Position l -> pp f "~%s:%s" l "[%src_pos]"
```

Comment:
I should put this string somewhere nice instead of hard-coding it left and right. Not sure where exactly